frontend/src/features/turnstile/useTurnstile.tsx

```diff
@@ -19,26 +20,6 @@ interface UseTurnstileProps extends TurnstileBaseConfig {
   enableUsage: boolean
 }
 
-type TurnstileConfig = TurnstileBaseConfig & {
```

Comment:
Removed since react-turnstile deps includes TS typings, which conflict with the existing self-defined type